src/components/Table/head/header.js

```diff
@@ -149,7 +149,7 @@ Header.propTypes = {
     onDeselectAllRows: PropTypes.func,
     tableId: PropTypes.string.isRequired,
     maxRowSelection: PropTypes.number,
-    bulkSelection: PropTypes.oneOf(['none', 'some', 'all']),
+    bulkSelection: PropTypes.oneOf([true, false, 'indeterminate']),
```

Comment:
In order to not make too many changes, we can use a map instead of change all this values